lib/browser/tag/util.js

```diff
@@ -136,7 +136,7 @@ function moveChildTag(tag, tagName, newPos) {
  */
 function initChildTag(child, opts, innerHTML, parent, skipName) {
   var tag = new Tag(child, opts, innerHTML),
-    tagName = getTagName(opts.root, skipName),
+    tagName = opts.tagName || getTagName(opts.root, skipName),
```

Comment:
@ogueg not sure about the `skipName` but I think it is irrelevant here since I am passing the dynamic name, but maybe I need to check for it?